src/tests/test_jitter_rng.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+* (C) 2024 Planck Security S.A.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include <vector>
+
+#include <botan/build.h>
+
+#ifdef BOTAN_HAS_JITTER_RNG
+
+   #include <botan/auto_rng.h>
+   #include <botan/entropy_src.h>
+   #include <botan/jitter_rng.h>
+   #include <botan/system_rng.h>
+
+   #include "tests.h"
+
+namespace Botan_Tests {
+
+std::vector<Test::Result> test_jitter_rng() {
```

Comment:
While working on the CI: with raised warning levels this doesn't compile, because of a warning:

```
  /home/runner/work/botan/botan/source/src/tests/test_jitter_rng.cpp:22:27: error: no previous declaration for ‘std::vector<Botan_Tests::Test::Result> Botan_Tests::test_jitter_rng()’ [-Werror=missing-declarations]
     22 | std::vector<Test::Result> test_jitter_rng() {
  ccache g++ -fsanitize=address -fsanitize=undefined -fno-sanitize-recover=undefined -fstack-protector -m64 -pthread -std=c++20 -D_REENTRANT  -O1 -fno-optimize-sibling-calls -fno-omit-frame-pointer -Wall -Wextra -Wpedantic -Wstrict-aliasing -Wcast-align -Wmissing-declarations -Wpointer-arith -Wcast-qual -Wzero-as-null-pointer-constant -Wnon-virtual-dtor -Wold-style-cast -Wsuggest-override -Wshadow -Wextra-semi -Werror -Wno-error=strict-overflow -Wno-error=zero-as-null-pointer-constant -Wno-error=non-virtual-dtor -Wno-error=maybe-uninitialized -Wno-error=stringop-overread -Wno-error=stringop-overflow -Wno-error=free-nonheap-object -Wno-error=restrict -DBOTAN_IS_BEING_BUILT -msse2 -I /home/runner/work/botan/botan/build/build/include/public -I /home/runner/work/botan/botan/build/build/include/internal  -isystem /home/runner/work/botan/botan/build/build/include/external -c /home/runner/work/botan/botan/source/src/tests/test_simd.cpp -o /home/runner/work/botan/botan/build/build/obj/test/test_simd.o
        |                           ^~~~~~~~~~~~~~~
  cc1plus: all warnings being treated as errors
```

Essentially, you'll have to wrap this function into an anonymous namespace.

Reply:
Done.